__tests__/Unit/hooks/statusApi.test.tsx

```diff
@@ -0,0 +1,58 @@
+import { setupServer } from 'msw/node';
+import { useGetIdleStatusQuery, useGetAllStatusQuery } from "@/app/services/statusApi";
+import handlers from '../../../__mocks__/handlers';
+import React, { PropsWithChildren } from 'react';
+import { act, renderHook } from '@testing-library/react-hooks';
+import { Provider } from 'react-redux';
+import { store } from "@/app/store";
+
+const server = setupServer(...handlers);
+
+beforeAll(() => {
+  server.listen();
+});
+afterEach(() => server.resetHandlers());
+afterAll(() => server.close());
+
+function Wrapper({ children }: PropsWithChildren<{}>): JSX.Element {
+  return <Provider store={store()}>{children}</Provider>;
+}
+
+describe('useGetIdleStatusQuery', () => {
+  test('returns idle status users', async () => {
+    const { result, waitForNextUpdate } = renderHook(() => useGetIdleStatusQuery("IDLE"), {
+      wrapper: Wrapper,
+    });
+    const initialResponse = result.current;
+    expect(initialResponse.data).toBeUndefined();
+    expect(initialResponse.isLoading).toBe(true);
+
+    await act(() => waitForNextUpdate());
+    
+    const nextResponse = result.current;
+    expect(nextResponse.data).not.toBeUndefined();
+    expect(nextResponse.data?.message).toEqual('Idle user status returned successfully')
+    expect(nextResponse.isLoading).toBe(false);
+    expect(nextResponse.isSuccess).toBe(true);
```

Comment:
Please write assert for checking users status is `IDLE`.

Reply:
done